core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -1414,7 +1420,8 @@ class ConnectionQuotas(config: KafkaConfig, time: Time, metrics: Metrics) extend
 
   class ListenerConnectionQuota(lock: Object, listener: ListenerName) extends ListenerReconfigurable {
     @volatile private var _maxConnections = Int.MaxValue
-    val connectionRateSensor = createConnectionRateQuotaSensor(Int.MaxValue, Some(listener.value))
+    val connectionRateSensor: Sensor = createConnectionRateQuotaSensor(Int.MaxValue, Some(listener.value))
+    val connectionRateThrottleSensor: Sensor = createConnectionRateThrottleSensor()
```

Comment:
nit: Do we really need to provide the type here? It seems that we usually don't provide it in that file.

Reply:
I added type because style check suggested (and also checked style guide) to annotate public members. However, I will change this to package scope, since we only need an outer class to access them.